src/inject/GPMInject/interface/mini.js

```diff
@@ -16,7 +16,7 @@ window.wait(() => {
     Emitter.fire('mini', { state: true });
     oldSize = remote.getCurrentWindow().getSize();
     mainWindow.setSize(MINI_SIZE, MINI_SIZE);
-    mainWindow.setMaximumSize(MINI_SIZE, MINI_SIZE);
```

Comment:
Hehe, sorry for the radio silence. Don't want to be sharing company secrets from my work machine! (Nothing terribly important though. ;-p)

I'm all about options, it'd be good for those of us that "need" or would like it to be adjustable.

Just curious, is #390 the fix for tiling issues mentioned in #351 ?